viv_utils/emulator_drivers.py

```diff
@@ -654,8 +653,9 @@ def step(self):
         else:
             return does_fallthrough, branches
 
-    def run(self, va: int):
+    def run(self, va: int, stopva: int = None):
```

Comment:
or add a `run_to_va` here as well?

Reply:
i think this should be done via a monitor.

also, returning early from the FullCoverage driver may skip work thats still enqueued. so, i don't believe this works as intended.

Reply:
so move `UntilVAMonitor` to `EmulatorDriver` and add a prehook?